charts/nydus-snapshotter/values.yaml

```diff
@@ -0,0 +1,72 @@
+# nydus-snapshotter Helm Chart Values
+
+# -- Override nydus-snapshotter name
+nameOverride: ""
+# -- Override nydus-snapshotter fullname
+fullnameOverride: ""
+
+nydusSnapshotter:
+  # -- nydus-snapshotter name
+  name: nydus-snapshotter
```

Comment:
`name` should be removed as there is only one nydusSnapshotter service, refer to https://github.com/bitnami/charts.